internal/integration/mysql_test.go

```diff
@@ -636,6 +811,20 @@ func (t *myTest) posts() *schema.Table {
 	return postsT
 }
 
+func (t *myTest) valueByVersion(v56 string, v57 string, v8 string) string {
+	switch t.version {
+	case "56":
+		return v56
+	case "57":
+		return v57
+	case "8":
+		return v8
+	default:
+		t.Fail()
```

Comment:
Maybe Fatal/Error with a message? 

```suggestion
		t.Fatalf("unrecognized version: %q", t.version)
```

Reply:
thanks, updated